src/D2L.Security.OAuth2/Keys/Default/D2LCryptoProviderFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.IdentityModel.Tokens;
+
+namespace D2L.Security.OAuth2.Keys.Default {
+	internal sealed class D2LCryptoProviderFactory : CryptoProviderFactory {

```

Comment:
`SecurityKey` as a `CryptoProviderFactory` property, but it's not overridable, so this needs to be provided to `SigningCredentials` / `TokenValidationParameters`